app/assets/stylesheets/styles/dashboard.scss

```diff
@@ -0,0 +1,22 @@
+// Place all the styles related to the dashboard controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+nav .nav-wrapper {
+  padding-right: 30px;
+  i {
+    font-size: 1.2rem;
+  }
+}
+
+.user_info {
+  background-color: #fff;
+  min-height: 30em;
```

Comment:
Properties should be ordered background-color, -moz-border-radius, -ms-border-radius, -webkit-border-radius, border-radius, margin, min-height, padding, width